pennylane_lightning/core/src/measurements/tests/Test_MeasurementsBase.cpp

```diff
@@ -694,6 +758,9 @@ template <typename TypeList> void testTensorProdObsExpvalShot() {
         using PrecisionT = typename StateVectorT::PrecisionT;
         using ComplexT = StateVectorT::ComplexT;
 
+#ifdef PL_USE_LAPACK
+        using MatrixT = std::vector<ComplexT>;
+#endif
```

Comment:
Apparently, this typedef definition is being used only once, in the block starting at line 845.
In this case, can we have this definition moved there?

Reply:
Let me move it.